README.md

```diff
@@ -33,20 +33,30 @@ Please note that the [plugin ecosystem in Prettier](https://prettier.io/docs/en/
 
 Plugin for Prettier to format pug code
 
-You can disable code formatting for a particular code block by adding <nobr>`<!-- prettier-ignore -->`</nobr> before ` ```pug `.
+You can disable code formatting for a particular code block by adding <nobr>`//- prettier-ignore`</nobr> comments inside your pug templates
+or by adding <nobr>`<!-- prettier-ignore -->`</nobr> before ` ```pug `.
 
 ````markdown
 Pug code with custom formatting:
 
-<!-- prettier-ignore -->
```

Comment:
This was part of the example ^^'
The example for how to use it in `md`

Reply:
Let's keep the discussion in the thread above with the rest of the context?

Reply:
I've pushed a new version of the README with its own chapter for Selectively ignoring automatic formatting